partner/partner-management-service/src/main/java/io/mosip/pms/partner/response/dto/NotificationsResponseDto.java

```diff
@@ -0,0 +1,21 @@
+package io.mosip.pms.partner.response.dto;
+
+import lombok.Data;
+import java.time.LocalDateTime;
+
+@Data
+public class NotificationsResponseDto {
+
+    private String notificationId;
+
+    private String notificationPartnerId;
+
+    private String notificationType;
+
+    private String notificationStatus;
+
+    private LocalDateTime createdDateTime;
+
+    private String notificationDetails;
```

Comment:
notificationDetails should not be String
add proper DTO's, we have already added in Batch Job, move to pms commons and use those